.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,16 +1,40 @@
-#### What does this PR do?
-#### Any background context you want to provide?
 #### What ticket does this PR close?
 Connected to [relevant GitHub issues, eg #76]
-#### Where should the reviewer start?
-#### How should this be manually tested?
-#### Screenshots (if appropriate)
-#### Has the Version and Changelog been updated?
-#### Questions:
-> Does this work have automated integration and unit tests?
 
-> Can we make a blog post, video, or animated GIF of this?
+## Checklist
+### Code
+- [ ] The code works.
+- [ ] The code performs its intended function and the logic is correct.
+- [ ] The code easy to understand.
+- [ ] Redundant or duplicate code has been removed.
+- [ ] All debugging code has been removed.
+- [ ] The code is as modular as possible.
```

Comment:
This is fine, but I know for myself, it would be helpful to perhaps also point to a design guide that helps elaborate what it means to be modular. For example:

"The code adheres to the [SOLID principles](https://en.wikipedia.org/wiki/SOLID)."